src/dispatch/database/core.py

```diff
@@ -247,7 +264,13 @@ def refetch_db_session(organization_slug: str) -> Session:
 @contextmanager
 def get_session() -> Session:
     """Context manager to ensure the session is closed after use."""
-    session = SessionLocal()
+    schema_engine = engine.execution_options(
```

Comment:
We should encapsulate this logic in a function that takes an org slug as a param, so we don't need to duplicate the code in `get_organization_session()`